build-system/compile/compile.js

```diff
@@ -46,9 +46,19 @@ let inProgress = 0;
 const MAX_PARALLEL_CLOSURE_INVOCATIONS =
   parseInt(argv.closure_concurrency, 10) || cpus().length;
 
-// Compiles AMP with the closure compiler. This is intended only for
-// production use. During development we intend to continue using
-// babel, as it has much faster incremental compilation.
+/**
+ * Compiles AMP with the closure compiler. This is intended only for
+ * production use. During development we intend to continue using
+ * babel, as it has much faster incremental compilation.
+ *
+ * TODO(#28387) set the correct types here.
+ * @param {*} entryModuleFilename
+ * @param {*} outputDir
+ * @param {*} outputFilename
+ * @param {*} options
+ * @param {*} timeInfo
```

Comment:
Pretty sure these are (in order): `string`, `string`, `string`, `Object`, and `Object`. (You'll need to figure out the shapes of the two objects.)

Reply:
Thanks, I think I've got the full types now.